### ISSUE TITLE
We Need Your Help

### ISSUE THREAD

**Issue by D19A66B55** [billthefarmer/editor#198]
F-Droid is a hopeless case.
We're still waiting to dl v1.84 of Editor & I wonder when that will take place.
Version 1.83 still at F-Droid & who knows how long we'll have to wait for Notes v1.34 to appear at F-Droid.
I don't understand why you can't just let us dl the releases from Github & waiting for F-Droid is totally annoying.
I would think you'd want to make it easy for us to use your products.
Well, have a nice day.

**Comment by billthefarmer** [comment]
They built it a couple of days ago. It will probably be in the next update. I don't build my apps, if I did you wouldn't be able to update via F-Droid without uninstalling and reinstalling. Sometimes they take a couple of days, sometimes longer. I did have a nice day, but it was a bit wet here.

**Comment by D19A66B55** [comment]
What do u mean>They built it a couple of days ago. It will probably be in the next update.<
I find F-Droid almost useless.
I've tried their app twice & uninstalled it. Updates don't seem to be a concern to them. I'm not the only person who complains about this issue.

**Comment by billthefarmer** [comment]
https://monitor.f-droid.org/builds/log/org.billthefarmer.editor/184

**Comment by D19A66B55** [comment]
Other authors use F-Droid & have releases on Github.
Is what you're saying is that you don't build your apps so you can't put the releases on Github?
I'm trying to understand the method of this madness🙃

**Comment by billthefarmer** [comment]
I wait for them to build it, then put it on the release. Then there's only one version of each release out there. There are other FOSS app stores, but, AFAIK, they get the apps from F-Droid.

**Comment by D19A66B55** [comment]
On 03/13/2023 I got my notification about Editor. It is now 03/26/2023.
Pretty ridiculous waiting this long for an update.
Actually, I can't remember ever waiting this long for an update except for the authors that don't have the releases on Github.
Usually, it's within the hour or less that releases are on Github by the authors who do it this way.
Besides the users, I don't really understand why any author would put up with this waiting game.
Well, I guess if I don't want to continue to be annoyed, I'll just get along with the versions I already have.
I do have better things to do than check, check, check with F-Droid to see if they've made any headway.

**Comment by billthefarmer** [comment]
I don't understand the mad tearing rush. There isn't anything magick in the latest version &ndash; I made all the black theme text white and the white theme text black so there isn't any grey, found a better way to create the icons for shortcuts so they're not blurred on a tablet and removed some complicated code that updates the highlighting.

**Comment by VimBowen** [comment]
I think this person saw a video by this guy, stating that fdroid is insecure. The video advised to use an RSS reader instead to receive notifications about app updates (hence why the author of this issue was notified about the release of your app.) the video goes on to explain that you can then grab the apk file from the releases page on the relevant repositories.

Having recently bought an android tablet, during my research I came across this video, as well as seeing a post on 'privacy guides' also advising against using fdroid unless absolutely neccessary. I am surprised by how android app repositories I have taken an interest in recently seem to have at least one comment relating to this 'issue.'

**Comment by billthefarmer** [comment]
Although F-Droid can be slow, especially when someone's app breaks their system, they scrupulously check the sources and build on an isolated virtual box. And they do reproducible builds of some apps. On the other hand there is all sorts of uncheckable stuff in GitHub releases with no sources.

**Comment by VimBowen** [comment]
Pretty much what I figured. Whilst I don't know exactly how f-droid works, and maybe arguments can be made that they are less than perfect - I don't see much difference between trusting fdroid vs trusting the arch team to build the packages I use on my PC.

I momentarily considered the method I previously mentioned, and when I couldn't even find a checksum for some apps let alone the signature key to verify the app, I figured I'll take my 'chances' with fdroid - so far so good.

Anyway just thought I'd let you know, take care 👋
